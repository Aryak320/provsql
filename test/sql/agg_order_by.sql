\set ECHO none
\pset format unaligned
SET search_path TO provsql_test, provsql;

CREATE TABLE agg_order_result AS
  SELECT city, count(*)
    FROM personnel
    GROUP BY city ORDER BY city;

SELECT remove_provenance('agg_order_result');

SELECT * FROM agg_order_result ORDER BY city;

DROP TABLE agg_order_result;

SELECT city, count(*)
FROM personnel
GROUP BY city ORDER BY count(*);
