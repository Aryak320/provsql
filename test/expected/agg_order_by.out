\set ECHO none
remove_provenance

(1 row)
city|count
Berlin|2 (*)
New York|2 (*)
Paris|3 (*)
(3 rows)
ERROR:  ORDER BY on the result of an aggregate function is not supported by ProvSQL
